packages/lesswrong/components/common/TabNavigationMenu/SubforumsList.tsx

```diff
@@ -19,58 +21,90 @@ const styles = ((theme: ThemeType): JssStyles => ({
     fragmentName: 'TagSubforumSidebarFragment',
     enableTotal: false,
     fetchPolicy: 'cache-and-network',
   })
-  
-  if (!results || !results.length) return <></>
-  
+
   // MenuItem takes a component and passes unrecognized props to that component,
   // but its material-ui-provided type signature does not include this feature.
   // Cast to any to work around it, to be able to pass a "to" parameter.
   const MenuItemUntyped = MenuItem as any
-  
-  const { TabNavigationSubItem } = Components
 
-  
+  const { TabNavigationSubItem, Loading, LWTooltip } = Components
+
   return (
     <span>
       <AnalyticsContext pageSubSectionContext="menuSubforumsList">
         <div>
-          <div className={classes.title}>Subforums</div>
-          {results.map((subforum) => (
-            <MenuItemUntyped
-              key={subforum._id}
-              onClick={onClick}
-              component={Link}
-              to={tagGetSubforumUrl(subforum)}
-              classes={{ root: classes.menuItem }}
-            >
-              <TabNavigationSubItem className={classes.subItem}>
-                {subforum.name}
-              </TabNavigationSubItem>
-            </MenuItemUntyped>
-          ))}
+          <MenuItemUntyped
+            key={"wiki"}
+            onClick={onClick}
+            component={Link}
+            to={`/${taggingNamePluralSetting.get()}/all`}
+            classes={{ root: classes.menuItem }}
+          >
+            <TabNavigationSubItem className={classes.subItem}>
+              <LWTooltip
+                title={`A sorted list of ${taggingNamePluralCapitalSetting.get()}, with wiki-style navigation`}
+              >
+                EA Wiki
+              </LWTooltip>
+            </TabNavigationSubItem>
+          </MenuItemUntyped>
+          {loading && !results?.length && <Loading />}
+          {!!results?.length && (
+            <>
+              <div className={classes.subforumsHeader}>Subforums</div>
+              {results?.map((subforum) => (
+                <MenuItemUntyped
+                  key={subforum._id}
+                  onClick={onClick}
+                  component={Link}
+                  to={tagGetSubforumUrl(subforum)}
+                  classes={{ root: classes.menuItem }}
+                >
+                  <TabNavigationSubItem className={classes.subItem}>
+                    <div className={classes.indented}>{startCase(subforum.name)}</div>
+                  </TabNavigationSubItem>
+                </MenuItemUntyped>
+              ))}
+              <MenuItemUntyped
+                key="all-subforums"
+                onClick={onClick}
+                component={Link}
+                to={"/search?contentType=Tags&query=&toggle%5BisSubforum%5D=true&page=1"}
+                classes={{ root: classes.menuItem }}
+              >
+                <TabNavigationSubItem className={classes.subItem}>
+                  <div className={classes.indented}>(see all)</div>
```

Comment:
I don't love how this looks, but it also looks a bit strange without this when there is only one subforum (which will be the case for logged out users)